addons/mail/static/src/model/model_manager.js

```diff
@@ -740,7 +737,10 @@ class ModelManager {
                 if (k === 'constructor') {
                     return target[k];
                 }
-                const field = target.constructor.fields[k];
+                const fields = target.constructor.fields;
+                const field = Object.prototype.hasOwnProperty.call(fields, k)
+                    ? fields[k]
+                    : undefined;
```

Comment:
weird, all models have `.fields`, no?

Reply:
to handle `toString` and other magic/inherited properties that are not fields